src/DynamicRupture/FrictionLaws/RateAndState.h

```diff
@@ -437,11 +411,10 @@ class RateAndStateBase : public BaseFrictionLaw<RateAndStateBase<Derived>> {
                               (fabs(normalStress[pointIndex]) * dMuF[pointIndex]) -
                           1.0;
         // ratio
-        tmp3[pointIndex] = nr[pointIndex] / dNr[pointIndex];
+        real tmp3 = nr[pointIndex] / dNr[pointIndex];
```

Comment:
so the division is made double and the result cast to real?

Reply:
Yes, it turned out that I need this division in double precision.